rpc/apis.go

```diff
@@ -43,14 +41,23 @@ const (
 	apiVersion = "1.0"
 )
 
+// API describes the set of methods offered over the RPC interface
+type API struct {
```

Comment:
I think I would rather this API struct to be an extension of the `go-ethereum` one given that making sure that we stay align with their API is key for different integrations.